tools/otsdb-dump-restore/setup.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
```

Comment:
I believe all we need to extend the requirements for java sources to python as well. All py files need to have a copyright and some pythondocs where appropriate.
Also I don't see tests here. Could you please add some or add some guide that could be used to understand the reproduction steps. What you expect after doing certain steps.

Reply:
I've added a guide for two possible workflows. Is this what you mean?
Regarding the copyright and the pythondocs, let's discuss them with the team.

Reply:
yes, that's great now